public/css/dataland-details.css

```diff
@@ -107,11 +107,6 @@
 	background-color: #DFD;
 }
 
-tr:hover.dsrow
```

Comment:
This doesn't even do anything in Dataland, so maybe just move it back, or remove?

Reply:
I guess it is removed now.